### DISCUSSION TITLE
Col/row span

### DISCUSSION THREAD

**Discussion by Pweder3** [TomSchimansky/CustomTkinter#1851] [Q&A]
Is it possible like in tkinter to set a wigets size based of of its start location in grid and then also set its collum span as its called in tkinter.

`scrollable_frame.grid(row=1, column=8, padx=(20, 0), pady=(20, 0), sticky="nsew",colSpan = 3,rowSpan=3`

**Comment by onlyforpeace** [comment]
yes columnspan=3,rowspan=3 run for me...

**Reply by avalon60** [reply]
Absolutely, but you have a syntax error (missing closing parenthesis)
